packages/gestalt/src/SlimBanner.js

```diff
@@ -55,44 +56,37 @@ function HelperLink({ accessibilityLabel, href, onClick, target, text }: HelperL
   );
 }
 
-type PrimaryActionType = {|
-  accessibilityLabel: string,
-  disabled?: boolean,
-  href?: string,
-  label: string,
-  onClick?: ({|
-    event:
-      | SyntheticMouseEvent<HTMLButtonElement>
-      | SyntheticMouseEvent<HTMLAnchorElement>
-      | SyntheticKeyboardEvent<HTMLAnchorElement>
-      | SyntheticKeyboardEvent<HTMLButtonElement>,
-    dangerouslyDisableOnNavigation: () => void,
-  |}) => void,
-  rel?: 'none' | 'nofollow',
-  target?: null | 'self' | 'blank',
-|};
+type PrimaryActionType =
+  | {|
+      role: 'link',
+      accessibilityLabel: string,
+      disabled?: boolean,
+      href?: string,
```

Comment:
make sure in link aren't optional

`href?`

Reply:
DONE